cumulusci/tasks/bulkdata/load.py

```diff
@@ -101,7 +101,9 @@ def _load_mapping(self, mapping):
             self._load_record_types([mapping["sf_object"]], conn)
         mapping["oid_as_pk"] = bool(mapping.get("fields", {}).get("Id"))
         job_id, local_ids_for_batch = self._create_job(mapping)
-        result = self._wait_for_job(job_id)
+        result = self._wait_for_job(job_id, error_behaviour="return")
+        if result == "CompletedWithFailures":
+            errors = self.error_messages  # noQA   # Todo: what to do with these errors?
```

Comment:
This is just the `f"Failures detected: {num_failures}"` message. It was already logged in _wait_for_job and I don't think there's a need to do anything more with it. The individual record errors will be found and reported while processing the job results.

Reply:
The actual error handling (a bit counterintuitively) is in `_generate_results_id_map()`, which first consumes the downloaded results file.